### PR TITLE
Create 'quorum' queues in RabbitMQ

### PR DESCRIPTION
**Pull request by aaguilartablada**
It closes issue https://github.com/dapr/docs/issues/3685

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md [hunk 1]
```diff
@@ -409,6 +409,24 @@ client.PublishEvent(ctx, PUBSUB_NAME, TOPIC_NAME, []byte(strconv.Itoa(orderId)),
 
 {{< /tabs >}}
 
+## Use quorum queues
+
+By default Dapr creates 'classic' queues. To create 'quorum' queues:
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md]:
```suggestion
By default, Dapr creates `classic` queues. To create `quorum` queues, add the following metadata to your subscription:
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
small edit

**Comment by aaguilartablada** [comment]
changes applied. thanks @hhunter-ms !

**Review by hhunter-ms — APPROVED** [review]

**Comment by msfussell** [comment]
@aaguilartablada - The `queuetype` property needs to added to the Component format Metadata. The section you added is a Wow to example, but we need the overall reference in the table. Actually I now see this old PR. https://github.com/dapr/docs/pull/3193. Can you pull the information from #3193 into this PR?

**Comment by aaguilartablada** [comment]
> @aaguilartablada - The `queuetype` property needs to added to the Component format Metadata. The section you added is a Wow to example, but we need the overall reference in the table. Actually I now see this old PR. #3193. Can you pull the information from #3193 into this PR?

The PR https://github.com/dapr/docs/pull/3193 is bad documentation, as my PR https://github.com/dapr/components-contrib/pull/2816 only allows classic and quorum queues. It does not include support for stream. 

I think my documentation is OK as queue type is indicated at subscription level, not at component level.

**Comment by hhunter-ms** [comment]
resolved conflicts

**Comment by msfussell** [comment]
@aaguilartablada - Look at my comment here https://github.com/dapr/docs/pull/3193/files#r1326422581
We do need to include all the metadata options and you have added `queuetype`in this PR. So can we merge #3193 along with this PR? It is unfortunate there are two docs PRs for this one code update.

**Comment by aaguilartablada** [comment]
@msfussell let me insist, please. 

My PR https://github.com/dapr/components-contrib/pull/2816 does not include metadata for RabbitMQ pubsub component. It includes metadata for subscription the same way subscription contains metadata for routing key without including it in component. The comment you just mentioned is to document a pull request that was closed without merging. You can go to the issue associated https://github.com/dapr/components-contrib/issues/2544 and then you will see that the PR https://github.com/dapr/components-contrib/pull/2556 is closed. This PR tried to add metadata to pubsub component, right. But my solution, that was already merged, has another approach: configuring the queue type through subscription.

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md [hunk 2]
```diff
@@ -412,6 +412,24 @@ client.PublishEvent(ctx, PUBSUB_NAME, TOPIC_NAME, []byte(strconv.Itoa(orderId)),
 
 {{< /tabs >}}
 
+## Use quorum queues
+
+By default, Dapr creates `classic` queues. To create `quorum` queues, add the following metadata to your subscription:
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md]:
```suggestion
By default, Dapr creates `classic` queues. To create `quorum` queues, add the following metadata to your pub/sub [subscription]({{< ref subscription-schema.md >}})
```

Comment (msfussell) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md]: @aaguilartablada - Got it. Thanks for the clarification. I closed the other PR (https://github.com/dapr/docs/pull/3193_ and looked at the code in yours to see this change. Suggest adding this link to the subscription format

**Pull request merged**